.trivyignore

```diff
@@ -32,6 +32,3 @@ CVE-2022-42889
 # Suppression for h2 2.1.214 password on command line vulnerability
 #   can be suppressed as we only run h2 locally and not on build environments
 CVE-2022-45868
-# Suppression for spring-web 5.3.24 as bundled with spring boot
```

Comment:
we're using spring web 6 so isn't relevant (automatically suppressed by spring boot upgrade)